Woof/Woof/View/OwnerFlow/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -13,10 +13,26 @@ struct DetailPetSitterView: View {
                 iconName: .IconName.filledPerson,
                 text: viewModel.fullName
             )
+            .contextMenu {
```

Comment:
Should we implement `UIMenuController `/`UIEditMenuInteraction` here?

I'd expect to see the context menu I can see everywhere in the app.

Reply:
I want to leave such version because I didn't find suitable implementation in a reasonable time and I've added [issue](https://github.com/ios-course/ironfoudation-team-project/issues/155) to impove that in the future.